nhais/main.py

```diff
@@ -1,6 +1,6 @@
 import tornado.httpclient
 import utilities.integration_adaptors_logger as log
-from requests import healthcheck_handler
+from mhs_common.request import healthcheck_handler
```

Comment:
This should be `mhs_common.handlers`, shouldn't it?

Reply:
no, it's in "common" now as it's shared with NHAIS

Reply:
I agree, that there is still something wrong with this approach as we don't uniquely identify sub-dependency packages but let's merge this and then work on better solution

Reply:
ahh damn, yes you are right